schemas/dab.draft.schema.json

```diff
@@ -695,36 +695,34 @@
     },
     "autoentities": {
       "type": "object",
-      "description": "Auto-entity definitions for pattern matching",
+      "description": "Defines automatic entity generation rules for MSSQL tables based on include/exclude patterns and defaults.",
       "patternProperties": {
         "^.*$": {
           "type": "object",
-          "additionalProperties": false,
           "properties": {
             "patterns": {
               "type": "object",
               "description": "Pattern matching rules for including/excluding database objects",
-              "additionalProperties": false,
               "properties": {
                 "include": {
                   "type": "array",
-                  "description": "T-SQL LIKE pattern to include database objects (e.g., '%.%')",
+                  "description": "MSSQL LIKE pattern for objects to include (e.g., '%.%'). Null includes all.",
                   "items": {
                     "type": "string"
                   },
                   "default": [ "%.%" ]
                 },
                 "exclude": {
                   "type": "array",
-                  "description": "T-SQL LIKE pattern to exclude database objects (e.g., 'sales.%')",
+                  "description": "MSSQL LIKE pattern for objects to exclude (e.g., 'sales.%'). Null excludes none.",
                   "items": {
                     "type": "string"
                   },
                   "default": null
                 },
                 "name": {
                   "type": "string",
-                  "description": "Interpolation syntax for entity naming, must be unique for every entity inside the pattern",
+                  "description": "Entity name interpolation pattern using {schema} and {object}. Null defaults to {object}. Must be unique for every entity inside the pattern",
                   "default": "{object}"
```

Comment:
I notice `"additionalProperties": false` on multiple other properties - like template, health etc. Please go through each of the occurrences - and ensure this is really needed or not.